container-course.js

```diff
@@ -2,28 +2,40 @@ import { LitElement, html, css } from 'lit-element';
 
 import './input-course.js';
 import './h1-course.js';
-
+import './list-course.js';
 
 class ContainerCourse extends LitElement {
     static get properties() { 
         return { 
           header: { type: String },
+          row: { type: String },
         };
       }
 
 
     render() {
         return html`
-        <input-course @input-change="${(event) => this.handleInputEvent(event, '')}"></input-course>
+        
+        <input-course @input-pulse="${(event) => this.handleInputEvent(event, '')}" @input-change="${(event) => this.handleChangeEvent(event)}"></input-course>
```

Comment:
No entiendo muy bien que hace ahora un input en el componente container¿

Reply:
Es el input en el que se escribe para que se creen nuevas rows en la tabla. Sigue anidado desde el ejercicio anterior, me parecio interesante mantenerlo asi porque le añade complejidad

Reply:
Además el titulo de la issue lo dice "Made the list-course a list that recive all values from the **input-course**"

Reply:
no vi que era el input-course mine error :)